jspoon/src/main/java/pl/droidsonroids/jspoon/HtmlField.java

```diff
@@ -38,7 +38,7 @@
 
     private void setLocaleFromTag(String localeTag) {
         if (localeTag.equals(Selector.NO_VALUE)) {
-            locale = Locale.getDefault();
+            locale = Locale.ROOT;
```

Comment:
> Jest możliwość podania locale przez usera :)
👍 
> Jak przetestować getDefault()?
Wywołać `setDefault()` na pocz. testu. Najlepiej opakować to w rule junitową, która w finally przywraca poprzednie locale, inaczej kolejne testy będą mieć zmienione locale.
> (czy lepiej by było go genrować z użyciem getDefault())?
Opcja z zahardkodowanym konkretnym tekstem i `setDefault()` na konkretne locale jest lepsza.
Przy generowaniu z użyciem `getDefault()` mogą istnieć bugi wykrywalne tylko przy konkretnym bieżącym locale.